.github/workflows/build-prod.yml

```diff
@@ -0,0 +1,138 @@
+name: Build prod
+
+on:
+  push:
+    branches: [ prod ]
+
+jobs:
+  all-microservices:
+    name: Get all microservices
+    runs-on: ubuntu-latest
+    outputs:
+      microservices: ${{ steps.microservices.outputs.list }}
+      microservices-spaced: ${{ steps.microservices.outputs.list-spaced }}
+      branch: ${{ steps.other-params.outputs.branch }}
+      docker-tag: ${{ steps.other-params.outputs.docker-tag }}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 30
+
+      - id: microservices
+        run: npx @lomray/microservices-cli microservices-list
+
+  checks-compile:
+    name: Code checks & compile & npm publish
+    needs: all-microservices
+    runs-on: ubuntu-latest
+
+    env:
+      ONLY: ${{ needs.all-microservices.outputs.microservices-spaced }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - run: echo "//npm.pkg.github.com/:_authToken=${{ secrets.GITHUB_TOKEN }}" > ~/.npmrc
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '18.19.0'
+          cache: 'npm'
+
+      - name: Install global dependencies
+        run: npm ci
+
+      - name: Install microservices dependencies
+        run: npx @lomray/microservices-cli global-install
+
+      - name: Check microservice eslint
+        run: npx @lomray/microservices-cli lint
+
+      - name: Check microservice typescript
+        run: npx @lomray/microservices-cli ts-check
+
+      - name: Run microservices tests with coverage
+        run: npx @lomray/microservices-cli test --coverage
+
+      - name: Compile microservices
+        run: npx @lomray/microservices-cli build
+
+      # note: this also update package.json version
+      - name: Publish npm packages / create github release
```

Comment:
this step should set same version for all microservices

just make sure

Reply:
Locally it runs as well, but for the forked repo i got this: The local branch prod is behind the remote one, therefore a new version won't be published.